processor/deltatocumulativeprocessor/config.go

```diff
@@ -0,0 +1,13 @@
+package deltatocumulativeprocessor // import "github.com/open-telemetry/opentelemetry-collector-contrib/processor/deltatocumulativeprocessor"
+
+import (
+	"go.opentelemetry.io/collector/component"
+)
+
+var _ component.ConfigValidator = (*Config)(nil)
+
+type Config struct{}
```

Comment:
I think we'll want to have a few configs:

1. MaxStaleness
    * We don't want to accumulate forever. We'd eventually OOM.
    * So we want the ability to "forget" metrics if they haven't been touch in X period of time
2. Include / Exclude filters
    * I don't have strong opinions on this
    * It would just match the current cumulativetodeltaprocessor
    * You could argue that someone should just add the filterprocessor to the pipeline and achieve the same results

Reply:
yes, staleness is on the roadmap! will add it in a later PR.
include/exclude is discussed in https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/30479#issuecomment-1914650740, but it would be a later PR as well.

Reply:
@sh0rez, would be willing to create an issue to capture the staleness issue? We have multiple processors in the works now which appear to need the same design considerations and it would be good to make sure we have a common solution for them.